cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -122,7 +122,12 @@
   {% set log_level = pillar['api_server_test_log_level'] -%}
 {% endif -%}
 
-{% set params = address + " " + etcd_servers + " " + etcd_servers_overrides + " " + cloud_provider + " " + cloud_config + " " + runtime_config + " " + admission_control + " " + service_cluster_ip_range + " " + client_ca_file + basic_auth_file + " " + min_request_timeout -%}
+{% set enable_garbage_collector = pillar['enable_garbage_collector'] -%}
+{% if pillar['enable_garbage_collector'] is defined -%}
```

Comment:
I don't know salt that well, but do you really need both this if block and the line before it?